src/component.js

```diff
@@ -136,7 +136,12 @@ function renderComponent(component) {
 			newVNode,
 			oldVNode,
 			component._globalContext,
-			component._parentDom.ownerSVGElement !== undefined,
+			component._parentDom.namespaceURI == 'http://www.w3.org/2000/svg'
```

Comment:
wouldn't it be more size efficient to use the old check here, embedding these strings can be costly on bytes

Reply:
I don't think so, as we already have the string for `createElementNS()` (and it's unavoidable there)

Reply:
Using only `.namespaceURI` cuts down on a few bytes too.